NBXplorer/AnnotatedTransactionCollection.cs

```diff
@@ -204,9 +204,8 @@ public AnnotatedTransactionCollection(ICollection<TrackedTransaction> transactio
 			{
 				if (tx.Height is int)
 				{
-					if(tx.IsMature)
-						ConfirmedTransactions.Add(tx);
-					else
+					ConfirmedTransactions.Add(tx);
+					if (!tx.IsMature)
 						ImmatureTransactions.Add(tx);
```

Comment:
So now Confirmed transactions will contain Immature txns also?

Reply:
yes. I thought about how I am using ConfirmedTransaction right now when I use this API (for example in BTCPay).
In BTCPay I do `unconfirmedTransactions.Concat(confirmedTransactions)` to get the whole list. In this list, we expect to see the immature transactions. Imagine somebody call bitcoin core's `generatetoaddress` but doesn't see the transaction in the list. It would be surprising to him.

Reply:
I see, thats a valid point.